tests/unit/amo/components/TestLandingPage.js

```diff
@@ -68,97 +82,85 @@ describe('<LandingPage />', () => {
   });
 
   it('dispatches getLanding when results are not loaded', () => {
-    const addonType = ADDON_TYPE_EXTENSION;
-    const dispatch = sinon.stub();
-    const errorHandler = new ErrorHandler({ id: 'some-id', dispatch });
-    render({
-      dispatch,
-      errorHandler,
-      params: { visibleAddonType: visibleAddonType(addonType) },
-      resultsLoaded: false,
-    });
+    const { store } = dispatchClientMetadata();
```

Comment:
This tends to be so common that I think it's better to make a var and initialize it in `beforeEach()`.